### PR TITLE
searchPlusIcon

### PR DESCRIPTION
**Pull request by keanan**
Add search icon with plus sign

image

### CODE REVIEW

File: packages/react/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@heetch/flamingo-react",
-  "version": "1.15.2",
```
Comment (obedparla) [inline packages/react/package.json:3]: Shouldn't this be done after merging, and by Lerna automatically?

Comment (ugogo) [reply packages/react/package.json:3]: Yes @obedparla, we must not update manually any version 🙏

File: lerna.json [hunk 2]
```diff
@@ -3,5 +3,5 @@
   "packages": [
     "packages/*"
   ],
-  "version": "1.15.2"
+  "version": "1.15.3"
```
Comment (ugogo) [inline lerna.json:6]: Same here https://github.com/heetch/flamingo/pull/134/files#r411548056

File: packages/react/package.json (hunk 1 above)
Comment (keanan) [reply packages/react/package.json:3]: Yes you are right, thanks

**Pull request merged**